.vscode/extensions.json

```diff
@@ -7,6 +7,7 @@
 		"dbaeumer.vscode-eslint",
 		"esbenp.prettier-vscode",
 		"ms-azure-devops.azure-pipelines",
+		"xyc.vscode-mdx-preview",
```

Comment:
Allows previewing `.mdx` output in VSCode

Reply:
It's ok, but makes me wonder if we want a new VSCode workspace for docs, like we have for tree and build-tools.